[IMP] subscriptions: parent doc restructure (#10771)
odoo/documentation: content/applications/sales/subscriptions.rst

      products.

   .. card:: Upsell subscriptions
      :target: subscriptions/upselling

      Explore the various ways to upsell customers on their subscriptions.

   .. card:: Renew subscriptions
      :target: subscriptions/renewals

      Process subscription renewals automatically or manually, include additional products, upsell,
      and more.

   .. card:: Close subscriptions
      :target: subscriptions/closing

      Decide whether customers should self-manage their subscriptions, or restrict their ability to
      do so.

   .. card:: Automation rules
      :target: subscriptions/automatic_alerts

      Utilize custom automation rules to set up emails, create tasks for salespeople, and even send
      satisfaction surveys to evaluate their experience.

   .. card:: Scheduled actions
      :target: subscriptions/scheduled_actions

      Pre-configured processes allowing users to automate certain tasks within a database.

   .. card:: Subscription reports
      :target: subscriptions/reports

      Odoo provides a series of reporting pages, helping companies analyze how subscriptions are

>       performing.

Reviewer: **Suggestion only relevant if Line 67 revision is accepted.

To create a new subscription, click on :guilabel:`New` from the **Subscriptions** or the :doc:`Sales
</applications/sales>` app. Then, from the resulting quotation, either:

- Select a :doc:`subscription plan </applications/sales/subscriptions/plans>` to auto-populate the
  quotation instantly.
- Or, fill out the quotation normally, making sure to select a recurrence, an end date (if
  necessary), and add :doc:`recurrent products </applications/sales/subscriptions/products>`.

.. tip::
   Define different invoice and delivery addresses by enabling the :doc:`Customer Addresses
   </applications/finance/accounting/customer_invoices/customer_addresses>` feature.

.. _subscriptions/confirmation:

Confirmation
============

Send the quotation to the customer for confirmation by clicking :guilabel:`Send by Email`, or
confirm it immediately by clicking :guilabel:`Confirm`.

.. tip::
   Click :guilabel:`Customer Preview` to preview the customer's point-of-view their quotation. From
   there, customers can sign and pay it, and communicate with the company, if needed.

.. _subscriptions/automatic-payments:

Automatic payments
==================

> Require the customer to set an automatic payment method, and pre-pay the subscription's first

Reviewer: ```suggestion
Prompt the customer to set up an automatic payment method and pre-pay the subscription's first
```

Reviewer: Optional/stylistic: I've not seen "require" used much in the imperative so this gave me pause. Suggesting an alternative here, feel free to use or discard.

occurrence before they can confirm their quotation. To do so, go to the :guilabel:`Other Info` tab

> of the quotation form, and tick the :guilabel:`Online payment` checkbox. The, designate what

Reviewer: ```suggestion
of the quotation form, and tick the :guilabel:`Online payment` checkbox. Then, designate what
```

percentage of the total should be paid.

If you leave :guilabel:`Online payment` unchecked, the customer does not have to pre-pay to start
the subscription. This means the payment is *not* automatic, and the customer **must** pay each
invoice manually.

.. important::

>    If the online confirmation requires a pre-payment, the customer can **only** select the

Reviewer: ```suggestion
   If the online confirmation requires pre-payment, the customer can **only** select the
```

>    If the online confirmation requires a pre-payment, the customer can **only** select the

Reviewer: ```suggestion
   If the online confirmation requires a pre-payment, the customer can **only** select
```

Reviewer: Optional/stylistic suggestion for word economy
